### PR TITLE
Add gitlab CD integration

### PR DESCRIPTION
**Pull request by zabbal**
The veracode pipeline is intended to be run as part of schedule only.

Test run: https://main.gitlab.in.here.com/olp/edge/ota/connect/back-end/web-events/pipelines/101106

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    variables:
+      - $VERACODE_API_ID
+  image: advancedtelematic/gitlab-jobs:0.1.0
+  before_script:
+    # The latest wrapper version can be found in https://repo1.maven.org/maven2/com/veracode/vosp/api/wrappers/vosp-api-wrappers-java/
+    - wget -q -O veracode-wrapper.jar https://repo1.maven.org/maven2/com/veracode/vosp/api/wrappers/vosp-api-wrappers-java/${VERACODE_WRAPPER_VERSION}/vosp-api-wrappers-java-${VERACODE_WRAPPER_VERSION}.jar
+    - ./sbt package
+    - mv target/scala-*/web-events_*.jar ./scan.jar
+  script:
+    - java -jar veracode-wrapper.jar -vid ${VERACODE_API_ID} -vkey ${VERACODE_API_KEY}
```
Comment (jochenschneider) [inline .gitlab-ci.yml]: So this can't generate a HTML result?

**Comment by jochenschneider** [comment]
Is this running on GitLab already? I can't find it.

**Comment by lbonn** [comment]
Might consider this improvement before merging: https://github.com/advancedtelematic/ota-tuf/pull/243/files

File: .gitlab-ci.yml (hunk 1 above)
Comment (zabbal) [reply .gitlab-ci.yml]: What do you mean by "HTML result"? This script package stuff up and submit it to external service. The scan results are available from that service at some point eventually but are not part of the pipeline.

**Comment by zabbal** [comment]
> Might consider this improvement before merging: https://github.com/advancedtelematic/ota-tuf/pull/243/files

This can be made in a separate PR - there're changes required in the corresponding dockerfile and we also have to test it in gitlab to make sure it works as intended. And the net result would be exactly the same - just with fewer lines of code. So I think it make sense to get this merge and optimize it later on so we could review scan results earlier.

**Comment by lbonn** [comment]
When we merge this early, we'll have to re-open a PR here, adding to the noise and needing again more review time. It's a bit strange to not do the right thing directly when the solution is within reach.

**Pull request merged**